app/src/main/java/org/mozilla/fenix/collections/CollectionCreationView.kt

```diff
@@ -232,22 +232,8 @@ class CollectionCreationView(
                 interactor.onBackPressed(SaveCollectionStep.NameCollection)
             }, TRANSITION_DURATION)
```

Comment:
should we also remove the back delay? Or is that still useful for some transition that's happening with the keyboard?

Reply:
Hello @ekager, I also saw that but it is not reported as a bug yet and it doesn't look bad so I decided to let it as it is. What do you think?